mirror/create_mirror_apps.sh

```diff
@@ -0,0 +1,24 @@
+[[ -z ${MIRROR_OUTPUT_DIR} ]] && export MIRROR_OUTPUT_DIR=${PWD}/mirror-dist
+[[ -z ${MIRROR_PYTHON_DIR} ]] && export MIRROR_PYTHON_DIR=${PWD}/mirror-dist/mirror_python/packages
```

Comment:
you should not rely on MIRROR_PYTHON_DIR as if we define MIRROR_OUTPUT_DIR to /var/www/html/ then this script will failed with this error:
cp: cannot stat '/home/ubuntu/pnda/mirror/mirror-dist/mirror_python/packages/kafka-python-1.3.5.tar.gz': No such file or directory
as $PWD is where I build it. so could you remove MIRROR_PYTHON_DIR and only rely on MIRROR_OUTPUT_DIR for getting mirror_python/packages/kafka-python-1.3.5.tar.gz please?